newpotato/term_client.py

```diff
@@ -179,41 +180,145 @@ def upload_jsonl(self, fn):
         with open(fn) as f:
             for i, line in tqdm(enumerate(f)):
                 data = json.loads(line)
-                self.hitl.store_triplets_from_annotation(data)
+                self.get_triplets_from_annotation(data)
         console.print("[bold cyan]Done![/bold cyan]")
 
-    def annotate(self):
-        for sen in self.hitl.get_unannotated_sentences(random_order=True, max_sens=3):
-            for pred, args in self.get_annotation_for_sen(sen):
-                self.hitl.store_triplet(sen, pred, args)
+    def get_triplets_from_annotation(self, data: Dict[str, Any]):
```

Comment:
I would probably try to keep the `term_client` file thin and move functions like this to a `utils` file, so we could reuse it on other frontends as well.